cli/src/pcluster/constants.py

```diff
@@ -66,13 +66,17 @@
 LUSTRE = "LUSTRE"
 OPENZFS = "OPENZFS"
 ONTAP = "ONTAP"
+FILECACHE = "FILECACHE"
 
 FSX_LUSTRE = "FsxLustre"
 FSX_OPENZFS = "FsxOpenZfs"
 FSX_ONTAP = "FsxOntap"
+FSX_FILE_CACHE = "FsxFileCache"
 
 # https://docs.aws.amazon.com/fsx/latest/APIReference/API_DescribeVolumes.html#FSx-DescribeVolumes-request-VolumeIds.
 FSX_VOLUME_ID_REGEX = r"^fsvol-[0-9a-f]{17}$"
+# https://docs.aws.amazon.com/fsx/latest/APIReference/API_FileCacheCreating.html#FSx-Type-FileCacheCreating-FileCacheId:~:text=Pattern%3A-,%5E(fc%2D%5B0%2D9a%2Df%5D%7B8%2C%7D)%24,-Required%3A%20No
+FSX_FILE_CACHE_ID_REGEX = r"^(fc-[0-9a-f]{8,18})$"
```

Comment:
Congrats with this point.
The public docs declares as regexp `^(fc-[0-9a-f]{8,})$` (not upper bound limit to 18). 
You did a step forward here by setting the limit to 21-3=18 as it should be according to a more in depth reading of the documentation. 
Thank you!

Reply:
Thank you!